pkg/controllers/job/job_controller_plugins_test.go

```diff
@@ -35,7 +35,7 @@ func newFakeController() *Controller {
 	VolcanoClientSet := volcanoclient.NewSimpleClientset()
 	KubeClientSet := kubeclient.NewSimpleClientset()
 
-	controller := NewJobController(KubeClientSet, KubeBatchClientSet, VolcanoClientSet)
+	controller := NewJobController(KubeClientSet, KubeBatchClientSet, VolcanoClientSet, 1)
```

Comment:
add more workers so we may catch some race condition issue.

Reply:
changed